src/infra/HttpServiceImpl.ts

```diff
@@ -22,13 +22,17 @@ export class HttpServiceImpl implements HttpService {
   get<T>(
     url: string,
     queryParams: QueryParams,
+    clientSdk: Record<string, string>,
     authToken?: string,
   ): Promise<T> {
-    return this.fetch(url, queryParams, {
+    return fetch(url, queryParams, {
       method: RequestMethods.GET,
       mode: 'cors',
       credentials: 'include',
-      ...(authToken && { headers: { Authorization: `Bearer ${authToken}` }}),
+      headers: {
+        'Client-SDK': formatAsHttpHeader(clientSdk),
```

Comment:
In the spec, the header is all caps (`CLIENT-SDK`) does this header get capitalized automatically somewhere?

Reply:
http headers are case insensitive! if you view them in chrome dev tools they all get turned to lowercase. That being said maybe matching the spec is still nicer here?

Reply:
we could still update though

Reply:
`Client-SDK` is what is used in the slack discussions so I'm not sure which is the correct version

Reply:
Gotcha! Sounds like it doesn't matter either way.